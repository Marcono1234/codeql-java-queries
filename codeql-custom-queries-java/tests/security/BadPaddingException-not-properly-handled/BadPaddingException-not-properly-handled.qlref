./security/BadPaddingException-not-properly-handled.ql
