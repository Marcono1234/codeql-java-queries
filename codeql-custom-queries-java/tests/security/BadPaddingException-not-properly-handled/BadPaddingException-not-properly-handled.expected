| Test.java:22:9:22:37 | stmt | Catches BadPaddingException or supertype but might not handle it properly |
| Test.java:33:11:33:63 | stmt | Catches BadPaddingException or supertype but might not handle it properly |
| Test.java:45:9:45:27 | stmt | Catches BadPaddingException or supertype but might not handle it properly |
| Test.java:52:10:52:25 | testThrowsHiding | Has `throws` clause which hides that BadPaddingException is thrown |
| Test.java:57:10:57:26 | testThrowsHiding2 | Has `throws` clause which hides that BadPaddingException is thrown |
